prover/setup.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# This script sets up the necessary data needed by the CPU/GPU FRI prover to be used locally.
+
+GPU_FLAG=""
+if [ "$1" = "gpu" ]; then
+    GPU_FLAG='--features "gpu"'
```

Comment:
BTW - error: none of the selected packages contains these features: "gpu", did you mean: gpu?

you might want to remove the quotes from the gpu here.

Reply:
That's correct, Fixed the feature flag name here [11540ff](https://github.com/matter-labs/zksync-era/pull/258/commits/11540ff3323e655ae0ce6259cb29b2903b149885)